test/provisioning/axios.mock.js

```diff
@@ -0,0 +1,57 @@
+class AxiosMock {
```

Comment:
This is how easy a mock can be. It's basically a copy of the original thing without slow functionality.

- Save a response beforehand
- Call and get the saved response